### DISCUSSION TITLE
Betty and Bobby 👌

### DISCUSSION THREAD

**Discussion by PaulDablue** [community/community#53848] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello, 
 I put up a beat made with Betty and Bobby and the Cabernet distortion from hammondeggsmusic on my YouTube channel and I left a link in the description hope that's ok.

https://youtu.be/Nux0mMXD3b4

The Betty and Bobby combo is a lovely thing 👍

Paul

**Comment by queenofcorgis** [comment]
Hi @PaulDablue , we’re glad you’re here 😀

Thanks for sharing your YouTube channel with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by PaulDablue** [reply]
I honestly didn't think about self promotion, just sharing something I made with the developer and giving some feedback and praise.

I posted in a thread titled Show and Tell, what else is that for?

As for overwhelming the users, seriously?

Lastly, whatever.

**Comment by PaulDablue** [comment]
Hello,

 I honestly didn't see it as self promotion, I thought I'd just share
something I made with the developer and give him some feedback.

I couldn't seem to find a way of giving feedback directly to the developer.

I posted in a thread titled Show and tell, what else is that for? showing
and telling.

"We want to make sure there is space for users to ask questions without
overwhelming them with other conversations"

It's not exactly overflowing with users or their questions is it?

Anyway whatever.
